apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,15 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import { ProgressBar } from '@fluentui/react-progress';
+import { PauseAnimationDecorator, TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+import { StoryWright, Steps } from 'storywright';
 
 storiesOf('ProgressBar converged', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(PauseAnimationDecorator)
```

Comment:
The vr-tests are still showing all the container, it's not cropping to the testWrapper. I think it might be because the fixed width wrapper is going in first? try flipping them to see if that helps.
```suggestion
  .addDecorator(PauseAnimationDecorator)
  .addDecorator(TestWrapperDecoratorFixedWidth)
```